src/olympia/lib/settings_base.py

```diff
@@ -1137,9 +1137,6 @@ def JINJA_CONFIG():
     'olympia.editors.tasks.reject_rereview': {'queue': 'editors'},
     'olympia.editors.tasks.send_mail': {'queue': 'editors'},
 
-    # Files
-    'olympia.files.tasks.extract_file': {'queue': 'files'},
```

Comment:
cc @jasonthomas that he doesn't get any notifications once this lands that the `files` queue stays empty.